testsuite/mpitests/test_connect_array_fixed_outdegree_mpi.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
Since this cannot be run automatically, move it to the `manualtests` directory. With #471 @jougs added a way allowing more complex tests requiring output files. Once that goes "mainstream", we can use it for this test as well.